pkg/proxyagent/agent/manifests/charts/addon-agent/templates/addon-agent-deployment.yaml

```diff
@@ -38,7 +38,7 @@ spec:
           args:
             - --proxy-server-host={{ .Values.serviceEntryPoint }}
             - --proxy-server-port={{ .Values.serviceEntryPointPort }}
-            - --agent-identifiers={{ list (printf "host=%s" .Values.clusterName) (printf "host=%s.%s" .Values.clusterName .Release.Namespace) (printf "host=%s.%s.%s" .Values.clusterName .Release.Namespace .Values.serviceDomain) | join "&" }}
+            - --agent-identifiers={{ .Values.agentIdentifiers }}
```

Comment:
is this scalable, what if we have 100 services?

Reply:
It should be as ANP doesn't mention any limitation on the number of `agent-identifiers`.
But it also still worth to add a test case for this, I will note this down.